mllib/src/test/scala/org/apache/spark/ml/feature/VectorAssemblerSuite.scala

```diff
@@ -147,4 +147,12 @@ class VectorAssemblerSuite
       .filter(vectorUDF($"features") > 1)
       .count() == 1)
   }
+
+  test("SPARK-25371: VectorAssembler with empty inputCols") {
+    val inputDF = Seq(
+      (1, Vectors.dense(1.0, 2.0)), (2, Vectors.sparse(2, Array(1), Array(3.0)))).toDF("i", "v")
```

Comment:
@mgaido91 . This is the only difference from the original one?

Reply:
yes, because we have no inputDF for the whole class here